cmd/podman/containers/run.go

```diff
@@ -76,8 +76,10 @@ func runFlags(cmd *cobra.Command) {
 	detachKeysFlagName := "detach-keys"
 	flags.StringVar(&runOpts.DetachKeys, detachKeysFlagName, containerConfig.DetachKeys(), "Override the key sequence for detaching a container. Format is a single character `[a-Z]` or a comma separated sequence of `ctrl-<value>`, where `<value>` is one of: `a-cf`, `@`, `^`, `[`, `\\`, `]`, `^` or `_`")
 	_ = cmd.RegisterFlagCompletionFunc(detachKeysFlagName, common.AutocompleteDetachKeys)
+
 	if registry.IsRemote() {
 		_ = flags.MarkHidden("preserve-fds")
+		_ = flags.MarkHidden("pidfile")
```

Comment:
We do not hide the --conmon-pidfile should we?
./bin/podman-remote create --help | grep conmon-pidfile
      --conmon-pidfile string                    Path to the file that will receive the PID of conmon

Reply:
I'm really indecisive. Is it hard to specify the pidfile location when access remote Podman service?
Please correct me if i am wrong.

Reply:
I am fine with hiding this, I am suggesting that you also hide conmonpid.

Reply:
Neither make much sense to specifiy remotely, since users would expect access to them, which will not be available.

Reply:
I have hidden the conmon-pidfile too.